### ISSUE TITLE
Camera is fixed; need multiple positions

### ISSUE THREAD

**Issue by BeytoA** [BeytoA/3D-Car-With-Settingspanel#1]
Although it can be rotated, currently, the camera is fixed on the hood. A key could be assigned to put it at the back of the car.

**Comment by ee0pdt** [comment]
Good idea, I made a simple PR to allow for setting position in the editor, but the idea of preset positions is a good one.

**Comment by ee0pdt** [comment]
In Car:

![image](https://user-images.githubusercontent.com/6094917/89109956-c156b480-d43d-11ea-954a-57177f213b25.png)

Rear

![image](https://user-images.githubusercontent.com/6094917/89109963-d6334800-d43d-11ea-84f3-716c82f831b9.png)

Bird's Eye

![image](https://user-images.githubusercontent.com/6094917/89109987-0da1f480-d43e-11ea-8931-80fa10feb631.png)

Top Down

![image](https://user-images.githubusercontent.com/6094917/89109993-17c3f300-d43e-11ea-9733-b27ffa5a2a3a.png)
